libs/@hashintel/block-design-system/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@hashintel/block-design-system",
+  "version": "0.0.1",
+  "description": "Block Design System",
+  "bugs": {
+    "url": "https://github.com/hashintel/hash/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:hashintel/hash.git",
+    "directory": "libs/@hashintel/block-design-system"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "HASH",
+    "url": "https://hash.ai"
+  },
+  "main": "src/main.ts",
+  "types": "src/main.ts",
+  "scripts": {
+    "build-storybook": "storybook build",
+    "fix:eslint": "eslint --fix .",
+    "lint:eslint": "eslint --report-unused-disable-directives .",
+    "lint:tsc": "tsc --noEmit",
+    "prepublishOnly": "PACKAGE_DIR=$(pwd) yarn workspace @local/repo-chores exe scripts/prepublish.ts",
+    "postpublish": "PACKAGE_DIR=$(pwd) yarn workspace @local/repo-chores exe scripts/postpublish.ts",
+    "storybook": "storybook dev -p 6006",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@emotion/react": "11.9.0",
+    "@emotion/styled": "11.8.1"
+  },
```

Comment:
I know this is how the other `design-system` is set up, but does it make sense to have MUI a peer dependency but not the emotion ones as peer dependencies? They're all needed for MUI

Reply:
Makes sense, moved them to peer dependencies in d0513dd01c57e408d5271ceaf280b7f9085108b5 and c86ea42f0cf40475c3b50cfbeb42d63ea33b0134. Should `@emotion/cache` in `@hashintel/design-system` also be moved there?

Reply:
Yes, might as well.